sdk/servicebus/Azure.Messaging.ServiceBus/tests/Administration/ServiceBusManagementClientLiveTests.cs

```diff
@@ -101,7 +100,7 @@ public async Task BasicQueueCrudOperations(bool premium)
                 Status = EntityStatus.Disabled
             };
 
-            if (premium)
+            if (premium && _serviceVersion == ServiceBusAdministrationClientOptions.ServiceVersion.V2021_05)
```

Comment:
The version check gets repeated enough, I wonder if it would be useful to extract it into helper method somewhere so that it can be more easily tweaked as service versions evolve.

Reply:
I think we would just need to change it to >= as we will want this version hardcoded.